src/main/commands/commandManager.ts

```diff
@@ -38,7 +38,7 @@ export class CommandManager implements ExtensionComponent {
         this.registerCommand(context, 'jfrog.xray.showOutput', () => this.showOutput());
         this.registerCommand(context, 'jfrog.xray.refresh', () => this.doRefresh());
         // Local state
-        this.registerCommand(context, 'jfrog.issues.open.ignore', issue => this.doOpenUrlInBrowser(issue.ignoreUrl));
+        this.registerCommand(context, 'jfrog.issues.open.ignore', issue => CommandManager.doOpenUrlInBrowser(issue.ignoreUrl));
```

Comment:
I just wonder why is it `jfrog.issues.open.ignore`?

Reply:
the only thing that we are doing in the ignore rule command is to open the ignore Url.

Reply:
But now are not ignoring anything, right?